cdap-app-fabric/src/main/java/io/cdap/cdap/internal/sysapp/SystemAppEnableExecutor.java

```diff
@@ -111,17 +111,20 @@ private ApplicationWithPrograms deploySystemApp(Arguments arguments) throws Exce
     ArtifactSummary artifactSummary = arguments.getArtifact();
 
     KerberosPrincipalId ownerPrincipalId =
-      arguments.getOwnerPrincipal() == null ? null : new KerberosPrincipalId(arguments.getOwnerPrincipal());
+        arguments.getOwnerPrincipal() == null ? null
```

Comment:
indentation is off, please be sure to use the right settings in your IDE

Reply:
Pardon me.. I didn't change those lines.. intellij behaving weirdly. Will take a look at settings. Thanks!